renv/activate.R

```diff
@@ -2,11 +2,13 @@
 local({
 
   # the requested version of renv
-  version <- "1.0.3"
+  version <- "1.0.11"
```

Comment:
How much of the changes to the `activate.R` were implemented by you vs. auto-change d upon updating renv?

Reply:
They're entirely automatic when you bump a version.